site/content/docs/alerting-and-retries/retries.md

```diff
@@ -7,61 +7,107 @@ menu:
+#### How often should I retry?
 
-**For checks that run on lower frequencies**, e.g. once per hour, it makes sense to pick a strategy that retries more often,
-over longer stretches of time. Let's look at an example:
+**For checks that run on lower frequencies**, e.g. once per hour, it makes sense to pick a strategy that retries more often, over longer stretches of time. Let's look at an example:
 
 1. A check with a 1-hour frequency breaks at 13:00 CET.
 2. An exponential retry strategy with 3 retries and a 5-second interval is configured.
 3. This generates retries spaced apart by 5 seconds, 25 seconds, 125 seconds.
-4. Your check is now retried at about 13:00:05, 13:00:30 and 13:02:35. 
+4. Your check is now retried at about 13:00:05, 13:00:30 and 13:02:35.
 
-If in that ~3-minute period your app / system recovers, and this was just a fluke, you will not be alerted. If the issue persists,
-Checkly will alert you as per your alert settings.
+If in that ~3-minute period your app / system recovers, and this was just a fluke, you will not be alerted. If the issue persists, Checkly will alert you as per your alert settings.
 
-**For checks that run on higher frequencies**, e.g. once per minute, it makes sense to pick a strategy that retries less often,
-and spaced apart by shorter intervals.
+**For checks that run on higher frequencies**, e.g. once per minute, it makes sense to pick a strategy that retries less often, and spaced apart by shorter intervals.
 
 Of course, picking the right strategy depends on your use case, tolerance for intermittent failures, SLO levels and other factors.
 
-## Retry from the same region
+## Check retries: location strategy
 
-You can toggle if a check should be retried from the same location or not. 
-
-- If you enable this, a check that fails will be retried in the same location. 
-- If you disable this, a check that fails will be retried in a different location, selected from the location you selected for the check.
+You can decide if a check should be retried from the same location or not with the toggle: 
+**Always retry from the same location the check failed in**
+- If enabled, a check that fails will be retried in the same location. 
+- If disabled, a check that fails will be retried in a different location, from the locations your check runs at.
 
 There are some tradeoffs to consider:
 
 - Retrying from the same location makes sense if you care strongly about the uptime of your app in one specific location, compared to other locations.
 - Retrying from a different location makes sense, if you want to make sure your app is up in at least one location. 
+
+
+## Test sessions retries
+
+Sporadically failing tests may be caused by a variety of reasons such as issues with the underlying infrastructure, and sometimes, simply re-running the test is enough to make it succeed again.
+
+Use retries to reduce noise in your test session results while providing information about the retry attempts.
```

Comment:
I would consider a link to the test sessions page here, to provide context in case the reader isn't quite familiar with the differences between check runs and test sessions.